docs/getting-started/testing.md

```diff
@@ -2,7 +2,7 @@
 
 ## Integration
```

Comment:
Can you please update this to `Cypress` instead, since we're not using the "Integration" terminology anymore.